### PR TITLE
Base structure

### PR DESCRIPTION
**Pull request by Lagovas**
Here is folder structure according to discussed structure in our internal wiki `/w/acrang/docs/`

### CODE REVIEW

File: acra/getting-started/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: Getting started
+bookCollapseSection: true
+---
+qweqwe
```
Comment (vixentael) [inline acra/getting-started/_index.md:5]: qwewqweqwe :D

**Review by vixentael — APPROVED** [review]
`weight` property might be missing, but we'll figure it out

**Review by ZhmakAS — APPROVED** [review]

**Review by iamnotacake — APPROVED** [review]

File: acra/getting-started/_index.md (hunk 1 above)
Comment (Lagovas) [reply acra/getting-started/_index.md]: oops. removed

**Comment by storojs72** [comment]
Hey, guys! I will try to help with reviewing (if time will permit), when new documentation (especially crytography-related) will be appearing here

**Pull request merged**